### PR TITLE
feat(branch-utilities): added branchName and isPr override via ENV

### PR DESCRIPTION
**Pull request by patricksuter**

### CODE REVIEW

File: packages/branch-utilities/src/lib/types/sc-override-env-var.type.ts [hunk 1]
```diff
@@ -1,5 +1,11 @@
 export interface CustomScOverrideEnv {
-  /** stringified object of type {@link GitHubContext} */
+  /** flag to allow usage of master branch locally */
   // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents,@typescript-eslint/naming-convention
   SC_OVERRIDE: 'true' | any
+  /** flag to override branchName for getBranchInfo(), needs to be used in conjunction with SC_OVERRIDE_IS_PR */
+  // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents,@typescript-eslint/naming-convention
+  SC_OVERRIDE_BRANCH_NAME: string | any
+  /** flag to override isPr getBranchInfo(), needs to be used in conjunction with SC_OVERRIDE_IS_PR */
```
Comment (michaelwittwer) [inline packages/branch-utilities/src/lib/types/sc-override-env-var.type.ts:8]: ...with SC_OVERRIDE_BRANCH_NAME

File: packages/branch-utilities/src/lib/base.utils.ts [hunk 2]
```diff
@@ -43,7 +43,12 @@ export function createStageInfo(stage: string): StageInfo {
  */
 export function getBranchInfo(env: unknown, branchName?: string): BranchInfo {
   let isPr = false
-  if (isGithubWorkflow(env)) {
+
+  if (isFullBranchOverrideDefined(env)) {
+    // full branch name override via en vars SC_OVERRIDE_BRANCH_NAME and SC_OVERRIDE_IS_PR
```
Comment (michaelwittwer) [inline packages/branch-utilities/src/lib/base.utils.ts:48]: en -> env

**Review by michaelwittwer — CHANGES_REQUESTED** [review]
Implementation looks good. Just some typos.

File: packages/branch-utilities/src/lib/types/sc-override-env-var.type.ts (hunk 1 above)
Comment (patricksuter) [reply packages/branch-utilities/src/lib/types/sc-override-env-var.type.ts]: thx, I changed it

**Review by michaelwittwer — APPROVED** [review]

**Comment by michaelwittwer** [comment]
@patricksuter Anything blocking this from being merged?

**Pull request merged**